### ISSUE TITLE
New placeholders

### ISSUE THREAD

**Issue by PsychoLynx** [Zrips/Residence#342]
Hello!

We have used Residence on our server for a long time, and for a while now I've had some trouble getting relevant Residence information for use with other plugins. Currently I have to use methods to read the data directly from Residence's .yml files, which makes some key aspects of the server a bit unstable.

Some suggestions for placeholders include:

> %residence_<player>_<residence>.<world>.has_flag[<flagname>]%
 - To see a player's state for the specified flag in a specified residence in a specified world.

> %residence_<player>_<residence>.<world>.flags%
 - To list a player's flags and their states on a specified residence

> %residence_<residence>.<world>_coordinates%
- To return a specified Residence's x,y,z coordinates (e.g. 123,50,-180|150,90,-200)

> %residence_<residence>.<world>_X1/X2/Y1/Y2/Z1/Z2%
- To return a specified residence's specified set of coordinates

> %residence_<residence>.<world>_X1/X2/Y1/Y2/Z1/Z2%

Basically it would be really helpful to be able to get ALL the info from the <worldname>.yml-file more easily, but these I find are most pressing.
